rust/src/transformations/polars/make_agg_trans/mod.rs

```diff
@@ -0,0 +1,232 @@
+use crate::combinators::assert_components_match;
+use num::Zero;
+use opendp_derive::bootstrap;
+use polars::prelude::*;
+
+use crate::core::{
+    Function, Metric, MetricSpace, StabilityMap, Transformation,
+};
+use crate::domains::{ExprDomain, LazyFrameDomain, LazyGroupByContext, LazyGroupByDomain, ExprMetric};
+use crate::error::*;
+use crate::traits::TotalOrd;
+
+#[bootstrap(ffi = false)]
+pub fn make_agg_trans<T: AggTransformation>(
+    input_domain: LazyGroupByDomain,
+    input_metric: T::InputMetric,
+    transformations: Vec<T>,
+) -> Fallible<Transformation<LazyGroupByDomain, LazyFrameDomain, T::InputMetric, <T::OutputMetric as ExprMetric<LazyGroupByContext>>::InnerMetric>>
```

Comment:
This type signature scares me. It wants the inner metric of the output metric. I think this could be hidden by introducing a new associated type on AggTransformation. TODO: tinker